internal/controllers/machine/machine_controller_status.go

```diff
@@ -499,6 +585,39 @@ func setReadyCondition(ctx context.Context, machine *clusterv1.Machine) {
 	v1beta2conditions.Set(machine, *readyCondition)
 }
 
+func calculateDeletingConditionForSummary(machine *clusterv1.Machine) v1beta2conditions.ConditionWithOwnerInfo {
```

Comment:
Let's add a comment explaining why we compute a deleting condition for summary
(e.g. to drop granular details from the message